### PR TITLE
Extra tire

### PR DESCRIPTION
**Pull request by ocorella**

### CODE REVIEW

File: tests/e2e/carInfo/carInfo.spec.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+describe('Car Information page', function() {
+
+  it('should go to carInfo page', function() {
+    browser.get(browser.baseUrl + 'carInfo');
+    expect(browser.getCurrentUrl()).toEqual("http://localhost:8100/#/carInfo");
```
Comment (joserv2812) [inline tests/e2e/carInfo/carInfo.spec.js]: use browser.baseUrl in toEqual("http://localhost:8100/#/carInfo");

File: tests/e2e/carView/carView.spec.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+describe('Car View page', function() {
+
+  it('should go to carView page', function() {
+    browser.get(browser.baseUrl + 'carView');
+    expect(browser.getCurrentUrl()).toEqual("http://localhost:8100/#/carView");
```
Comment (joserv2812) [inline tests/e2e/carView/carView.spec.js]: use browser.baseUrl in toEqual

File: tests/e2e/login/login.spec.js [hunk 3]
```diff
@@ -1,38 +1,31 @@
 describe('Login', function() {
-  var username = element(by.model('LoginController.username'));
-  var password = element(by.model('LoginController.password'));
+  var username = element(by.model('vm.username'));
+  var password = element(by.model('vm.password'));
   beforeEach(function() {
-  	  browser.get('');	
-  	  browser.sleep(500); 
+      browser.get('');
```
Comment (joserv2812) [inline tests/e2e/login/login.spec.js]: fix indentation

File: tests/e2e/tireRevision/tireRevision.spec.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+describe('Tire Revision page', function() {
+
+  it('should go to tireRevision page', function() {
+    browser.get(browser.baseUrl + 'tireRevision');
+    expect(browser.getCurrentUrl()).toEqual("http://localhost:8100/#/tireRevision");
```
Comment (joserv2812) [inline tests/e2e/tireRevision/tireRevision.spec.js]: use browser.baseUrl in toEqual

File: tests/specs/feedback/feedback.ctrl.spec.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+/*describe('Feedback Controller', function() {
```
Comment (joserv2812) [inline tests/specs/feedback/feedback.ctrl.spec.js:1]: why this file is commented?

File: tests/specs/login/login.services.spec.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    });*/
+
+    beforeEach(inject(function(_$firebaseObject_, _LoginFirebaseService_) {
+      LoginFirebaseService = _LoginFirebaseService_;
+    }));
+
+    it('is defined', function() {
+        expect(LoginFirebaseService).toBeDefined();
+      });
+    /*it('fills new revision data ', function() {
```
Comment (joserv2812) [inline tests/specs/login/login.services.spec.js]: you can use xit to ignore the case

File: tests/specs/feedback/feedback.ctrl.spec.js (hunk 5 above)
Comment (ocorella) [inline tests/specs/feedback/feedback.ctrl.spec.js:1]: Johnny is going to make some changes in the controller, so I will wait to mock it properly

File: tests/e2e/carDeliveryInfo/carDeliveryInfo.spec.js [hunk 7]
```diff
@@ -1,20 +1,11 @@
 describe('Car Delivery Information page', function() {
 
   it('should go to carDeliveryInfo page', function() {
-    browser.get('http://localhost:8100/#/carDeliveryInfo');
-    expect(browser.getCurrentUrl()).toEqual("http://localhost:8100/#/carDeliveryInfo")
+    browser.get(browser.baseUrl + 'carDeliveryInfo');
+    expect(browser.getCurrentUrl()).toEqual("http://localhost:8100/#/carDeliveryInfo");
```
Comment (joserv2812) [inline tests/e2e/carDeliveryInfo/carDeliveryInfo.spec.js:5]: use browser.baseUrl

**Comment by joserv2812** [comment]
resolve conflicts

**Comment by joserv2812** [comment]
squash commits

**Pull request closed without merging**